### ISSUE TITLE
ccpUrl gets overwritten by redirect

### ISSUE THREAD

**Issue by alxx** [aws/amazon-connect-streams#94]
I'm using SAML for federated SSO, with an HTML page running the Streams API.

My init function:

```javascript
connect.core.initCCP(containerDiv, {
            ccpUrl: 'https://XXX.awsapps.com/connect/ccp#', 
            loginPopup: true, 
            softphone: {
                disableRingtone: true,
                allowFramedSoftphone: true
            }
        });
```

Here's what happens when I'm not logged in: the function runs, it uses the `ccpUrl` as the iframe source, but that URL returns a `Location` header:

> Location: /connect/login?landat=%2Fconnect%2Fccp

So now the iframe tries to use that as a source, and it fails because _Connect literally sent itself somewhere where it doesn't allow itself_:

> Refused to display 'https://XXX.awsapps.com/connect/login?landat=%2Fconnect%2Fccp#' in a frame because it set 'X-Frame-Options' to 'sameorigin'.

I have no idea what to try next. Does anyone have any ideas?

**Comment by 0-g-r-e-g-0** [comment]
There is a solution for this in issue [82](https://github.com/aws/amazon-connect-streams/issues/82)
By making the code changes that are discussed in that issue, I was also able to get SSO working.

**Comment by jagadeeshaby** [comment]
Code changes are available on 1.3.2 

Closing the issue. please feel free to re-open if the issue still persists.
